qlib/model/trainer.py

```diff
@@ -45,6 +45,37 @@ def begin_task_train(task_config: dict, experiment_name: str, recorder_name: str
     return recorder
 
 
+def fill_placeholder(config: dict, config_extend: dict):
```

Comment:
Check the format of the key of `config_extend` will make it safer (e.g. r"<[^<>]+>").

Reply:
ok